scripts/test/site_config.py

```diff
@@ -109,15 +109,17 @@ def __init__(self, definition_file):
             self.portbase = int(os.environ["PORTBASE"])
         self.port_offset = 100
         self.timeout = 1800
+        if "timeLimit".upper() in os.environ:
+            self.timeout = int(os.environ["timeLimit".upper()])
+        elif "timeLimit" in os.environ:
+            self.timeout = int(os.environ["timeLimit"])
```

Comment:
Why do we need to check for both the environment variables `timeLimit` and `TIMELIMIT` here?
Isn't one enough?

Reply:
Yes, it was like that before (I just moved the code block to a different place). I didn't want to change the logic because I was not sure whether this would have some unintended side effects. But AFAIK this script is only run from CircleCI, so it should probably be safe.